docs-en/chapter_array_and_linkedlist/list.md

```diff
@@ -857,13 +857,13 @@ After sorting the list, we can use algorithms often tested in array-related algo
 
 ## List Implementation
 
-Many programming languages have built-in lists, such as Java, C++, Python, etc. Their implementations are quite complex, with very meticulous settings for parameters such as initial capacity and expansion multiplier. Interested readers can refer to the source code for learning.
+Many programming languages come with built-in list structures, including Java, C++, Python, etc. Their implementations tend to be intricate, featuring carefully considered settings for various parameters, like initial capacity and expansion factors. Readers who are curious can delve into the source code for further learning.
```

Comment:
"built-in list structures" -> "lists"

I suggest removing the word 'structure' for brevity.